packages/account/src/Components/self-exclusion/self-exclusion-article.tsx

```diff
@@ -37,9 +39,9 @@ const SelfExclusionArticle = () => {
                           ),
                       ]
             }
-            {...((!is_appstore || isMobile()) && { onClickLearnMore: toggleArticle })}
+            {...(is_mobile && { onClickLearnMore: toggleArticle })}
```

Comment:
A question: what happens if `is_mobile` is `false`? Will it not break the spreading of nothing?

Reply:
javascript returns empty object in that case.